src/main/app/src/pages/ScheduleActivityPage.js

```diff
@@ -20,7 +20,8 @@ const ScheduleActivityPage = ({isLoggedIn, accessToken, userId, activity, links,
   // Event fields stored as component state.
   const [title, updateTitle] = React.useState("");
   const [startTime, updateStartTime] = React.useState(new Date());
-  const [endTime, updateEndTime] = React.useState(new Date());
+  // Set end date 30mins ahead.
+  const [endTime, updateEndTime] = React.useState(new Date(Date.now() + (60000 * 30)));
```

Comment:
Usually it is better to comment about a variable on the right. I remember commenting above, and I was told that commenting like this is better: So, it would be,
const [endTime, updateEndTime] = React.useState(new Date(Date.now() + (60000 * 30)));    // Set end date 30mins ahead.

Reply:
Gotcha. Fixed it.